xbmc/network/WebServer.cpp

```diff
@@ -974,7 +974,11 @@ int CWebServer::ContentReaderCallback(void *cls, size_t pos, char *buf, int max)
     return -1;
 
   if (g_advancedSettings.CanLogComponent(LOGWEBSERVER))
+#if (MHD_VERSION >= 0x00090200)
```

Comment:
Semi related, @wsnipex are there any platform that has mhd older than this? Got me thinking if we can get rid of a few ifdefs related to mhd